### DISCUSSION TITLE
Regarding Completion Status

### DISCUSSION THREAD

**Discussion by Sahil-Shimpi** [github/india#329] [General]
I just completed Describe Core Azure Concepts module of AZ-900. It is still showing status as 'In Progress'. I know that they had mentioned in the orientation session that it gets refreshed time to time. But, since I had an issue with Industry sessions as well, I'm kind of panicking. So please let me know if you're facing the same issue or it's just me.

**Comment by TheSuranaverse** [comment]
@Sahil-Shimpi From what I understand from orientation and reading the discussions, the self-learning tab will update once a week. So even if it shows "In progress" or "incomplete" continue doing the course in the order shown by the self-learning tab, and once the portal updates after a week, your completed courses should ideally be updated. If you see a refresh date and a course you completed has not been updated, that's when you should email support.

**Comment by TheSuranaverse** [comment]
@Sahil-Shimpi For more continuous collaboration & instant help
A group for all to get notified for any Official FRT Announcement, Discuss & Clear any Doubts, Learn & Complete together!
A one-stop place for daily Events/Workshops + Hackathons + Internships/Jobs/Free Scholarship & Courses + 24/7 Live Music & much more...

Link to join: https://discord.gg/ceMXzhfaka

Thank you 🙏

**Comment by Soumajit-Roy** [comment]
> 
> 
> I just completed Describe Core Azure Concepts module of AZ-900. It is still showing status as 'In Progress'. I know that they had mentioned in the orientation session that it gets refreshed time to time. But, since I had an issue with Industry sessions as well, I'm kind of panicking. So please let me know if you're facing the same issue or it's just me.

Login and Go to the Orientation section you can download the recorded session.
![fd](https://user-images.githubusercontent.com/91539827/136533301-4b4bfe04-920b-4b29-8212-e4c6bfba375f.jpg)
